coralproject/talk
# [CORL-1297] Move Comment Box

## What does this PR do?

With the `ALTERNATE_OLDEST_FIRST_VIEW` feature flag enabled, this pull request will move the comment form to the bottom of the stream as well as introduce some controls to scroll around and make navigation easier.



## What changes to the GraphQL/Database Schema does this PR introduce?

- Added `FEATURE_FLAG.ALTERNATE_OLDEST_FIRST_VIEW`



## How do I test this PR?



1. Enable the `ALTERNATE_OLDEST_FIRST_VIEW` feature flag.
2. On a comment stream, sort by "Oldest First"
3. See the new UI!
